stdpopsim/catalog/HomSap/demographic_models.py

```diff
@@ -1554,12 +1554,18 @@ def _africanBoyko():
     id = "Africa_1B08"
     description = "African population"
     long_description = """
-    This was the best fit model for the African population described
-    in Boyko et al (2008). It is a two-epoch instantaneous growth model
-    and values were extracted from Table S1 of the article.
+        African-American two-epoch instantaneous growth model from Boyko et al
+        2008, fit to the synonymous SFS for the 11 of 15 African Americans showing
+        the least European ancestry, using coalescent simulations with
+        recombination with the maximum likelihood method of Williamson et al 2005;
+        times were calibrated assuming 3e5 generations since human-chimp divergence
+        and fitting the number of synonymous human-chimp differences. Mutation and
+        recombination rates were assumed to be the same (1.8e-8).
     """
     populations = [
-        stdpopsim.Population(id="AFR_Boyko", description="African Boyko"),
+        stdpopsim.Population(
+            id="African", description="African-Americans from Boyko et al 2008"
```

Comment:
It feels weird to call this "african" when it's african-americans; what do you think?